.github/workflows/build.yml

```diff
@@ -397,11 +397,32 @@ jobs:
           name: session-build-${{ matrix.sys.os }}
           path: build/
 
+  scan:
```

Comment:
Please add the same `if` conditional as deploy. Which has it only run on nightly and release branches.

eg. 
```
if: contains(fromJSON('["refs/heads/master", "refs/heads/beta", "refs/heads/release", "refs/heads/LTS"]'), github.ref) || startsWith(github.event.pull_request.head.ref, 'version/')
```

Reply:
You missed my comment about this:

> I disagree that it should only run on release branches and nightlies. PRs should not introduce CVEs only for them to be caught later. We should be proactive. It took a whopping 5s to scan all generated binaries.

Until that is addressed, I am not okay with doing what you requested.